README.md

```diff
@@ -121,6 +122,16 @@ pip install super-gradients
 ```
 
 ## What's New
+
+__________________________________________________________________________________________________________
+Version 3.1.3 (July 19, 2023)
+
+* [Pose Estimation Task Support](https://docs.deci.ai/super-gradients/documentation/source/PoseEstimation.html) - Check out fine-tuning [notebook example](https://colab.research.google.com/drive/1NMGzx8NdycIZqnRlZKJZrIOqyj0MFzJE#scrollTo=3UZJqTehg0On) 
+* Pre-trained modified [DEKR](super_gradients/recipes/coco2017_pose_dekr_w32_no_dc.yaml) model for pose estimation (TensorRT-compatible)
```

Comment:
Link not working

Reply:
Fixed in dd3efe7de4b591f293b528f9a3ed1ad72ebcfbf2